haiku-games/deathchase3d/patches/deathchase3d-0.9.patchset

```diff
@@ -0,0 +1,165 @@
+From 76ac1d69a58ca0d39edf67307f2f71e837e13ae2 Mon Sep 17 00:00:00 2001
+From: Bach Nguyen <bach5000@gmail.com>
+Date: Sun, 11 Nov 2018 23:03:20 +0000
+Subject: [PATCH] Makefile + minor changes
```

Comment:
Why is this necessary when the source provides a `./configure` already?

Reply:
Will investigate... I honestly have no idea why I added a makefile :)
Maybe because I was afraid the generated makefile wouldn't have correct directories for Haiku.